maker/src/main.rs

```diff
@@ -1,14 +1,14 @@
 use anyhow::Result;
 use maker::logger;
 use maker::routes;
-use std::env::temp_dir;
+use std::env::current_dir;
 use std::time::Duration;
 use ten_ten_one::wallet;
 use tracing::metadata::LevelFilter;
 
 #[rocket::main]
 async fn main() -> Result<()> {
-    let path = temp_dir();
+    let path = current_dir()?.join("data").join("maker");
```

Comment:
much better :) 
Note: we will need to be able to pass this in as an arg once we want to run it in a container.

Reply:
yes, `$DATA_DIR` and `$NETWORK` should soon be added from `clap` :)